css/style.css

```diff
@@ -28,17 +28,23 @@ body::before
 }
 
 /* show signature download on hover */
-span.sig_btn > a:last-child
+span.sig_btn
 {
+    position: relative;
+    height: 4.4em;
```

Comment:
The download page isn't that performance sensitive ;), but yeah avoiding the hard-coded height would be nice.